### DISCUSSION TITLE
Subscribe Bookmarklet does not work for subscribing to YouTube playlist

### DISCUSSION THREAD

**Discussion by lamyergeier** [FreshRSS/FreshRSS#6398] [Q&A] [answered]
### Describe the bug

Example: [Medicine & Biology - YouTube](https://www.youtube.com/playlist?list=PLFs4vir_WsTyY31efyHdmtp9l7DpR0Wvi)

The FreshRSS message: `Blast! The newsfeed could not be added. [Check FreshRSS logs](http://[::1]:8095/i/?a=logs) for details. You can try force adding by appending #force_feed to the URL.`

### Logs

Unknown error for feed [https://www.youtube.com/playlist?list=PLDTPrMoGHssDgTkiSRzvXaxPkbxVkIiAC] 

### To Reproduce

1. Go to https://www.youtube.com/playlist?list=PLFs4vir_WsTyY31efyHdmtp9l7DpR0Wvi
2. Click on Subscribe bookmarklet

### FreshRSS version

1.23.1 

### Environment information

- Database version: SQLite
- Installation type:  Docker
- OS: Fedora 40
- Browser: Firefox 125

**Comment by math-GH** [comment]
from the php_error_log:
```
[27-Apr-2024 23:07:01 Europe/Berlin] 
PHP Notice: A feed could not be found at `https://www.youtube.com/playlist?list=PLFs4vir_WsTyY31efyHdmtp9l7DpR0Wvi`; 
the status code is `200` and content-type is `text/html; charset=utf-8` in ...\lib\SimplePie\SimplePie.php on line 1829
```

**Comment by lamyergeier** [comment]
Every Playlist from YouTube have a rss file and I was able to add it manually, but the bookmarklet is not working.

**Comment by math-GH** [comment]
The problem with Youtube (in general, it does not matter if it is a video, a channel nor a playlist) is that Youtube does not deliver RSS feeds by default. Just with a "trick". The RSS feed is not linked in the page that you visit, so FreshRSS cannot detect the feed.

That is the reason of f.e. the Youtube extension https://github.com/cn-tools/cntools_FreshRssExtensions

FreshRSS core does not have tweaks for single websites. Extension can solve this gap much better.

Ping @cn-tools : Is your extension ready for detecting RSS feeds of Youtube playlists via Bookmarklet?

**Comment by cn-tools** [comment, marked as answer]
Yes, my provided extension should work with YouTube playlist URLs (as i know/seen). A quick test with the URL above worked as expected.
